OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs

```diff
@@ -876,6 +876,30 @@ internal static void UpgradeActorRules(int engineVersion, ref List<MiniYamlNode>
 					if (node.Key.StartsWith("WithRotor"))
 						node.Value.Nodes.RemoveAll(p => p.Key == "Id");
 
+				if (engineVersion < 20150427)
+				{
+					// Change RenderFlare to RenderSprites + WithSpriteBody
+					if (node.Value.Nodes.Any(x => x.Key == "RenderFlare"))
+					{
+						node.Value.Nodes.Where(x => x.Key == "RenderFlare").Do(x => x.Key = "RenderSprites");
```

Comment:
The tree for the current actor. I don't think putting much effort in upgrade rules is worth the couple of milliseconds you win when you run the utility (*once!*).